introduction.md

```diff
@@ -1,3 +1,10 @@
 Introducing me, winehook, to this dojo.
 
-Also, my favorite CodeSchool path is: JavaScript.  Although I like them all!
\ No newline at end of file
+Also, my favorite CodeSchool path is: JavaScript.  Although I like them all!
+
+Deadly Skills:
+
+* PHP
+* databases
+* JavaScript (sort of)
+* git
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?